themes/default/content/docs/esc/environments.md

```diff
@@ -205,6 +205,30 @@ $ esc env get myorg/test greeting
   • test:10:15
 ```
 
+### Pulumi Contextual information
+
+Contextual information is available and can be used to interpolate on any environment value.
+
+```yaml
+values:
+  personalNamespace: ${context.rootEnvironment.name}/${context.pulumi.user.login}
+```
+
+It can be accessed through the `context` attribute and contains the following information:
+
+```yaml
```

Comment:
I think this one is slightly confusing. Because the users still need to assign a path name to use the Context. copy-pasting this into a YAML won't work every if I assign a path name.

Maybe remove this yaml and we can explain it via text?

Reply:
The idea was to show its structure and hierarchy. As we add more attributes to each object, will look really weird.

I just pushed the change, feel free to push changes